tasks/thor/example.rb

```diff
@@ -192,12 +199,14 @@ class Hello
       media_type V1::MediaTypes::Hello
       version '1.0'
 
-      routing do
-        prefix '/api/hello'
+      prefix '/api/hello'
+
+      # Will apply to all actions of this resource
+      action_defaults do
```

Comment:
You can do the same thing with just setting the trait on the resource directly, that'll show it properly at the resource level.